rules/puppet_st2_ci_el7.yaml

```diff
@@ -16,4 +16,5 @@
         ref: "st2cd.workroom_puppet_st2_ci"
         parameters:
             revision: "{{trigger.body.head_commit.id}}"
+            hostname: "st2wp-el7-{{trigger.body.head_commit.id | truncate(8, False, '')}}"
```

Comment:
@DoriftoShoes @lakshmi-kannan 
I follow @lakshmi-kannan path here using commit ID. Trouble is, re-running it on the same github hook... FORCES me to clean-up AWS zombies :) Keep? Change to using execution-id like @DoriftoShoes was doing?